server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java

```diff
@@ -160,6 +160,10 @@ private void setDefault(DbSession session, QualityProfileDto profile) {
     db.qualityProfileDao().update(session, profile.setDefault(true));
   }
 
+  /**
+   * @deprecated replaced by {@link org.sonar.server.qualityprofile.ws.QProfileWsSupport#getProfile(DbSession, org.sonar.server.qualityprofile.ws.QProfileRef)}
+   */
+  @Deprecated
   public QualityProfileDto find(DbSession dbSession, QProfileRef ref) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)